tss-esapi/src/interface_types/algorithm.rs

```diff
@@ -76,6 +76,52 @@ impl TryFrom<TPMI_ALG_HASH> for HashingAlgorithm {
     }
 }
 
+/// Provides the value of the digest used in this crate for the digest.
```

Comment:
I am not sure this is the best place to put this. This file is mainly containing the TPMI_ALG stuff.